http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2773,3 +3036,130 @@ components:
         labels:
           type: object
           description: The labels to be placed on the disk
+
+    CreateAppRequest:
+      description: the configuration of an app, used to create an app
+      type: object
+      properties:
+        diskConfig:
+          type: object
+          $ref: '#/components/schemas/PersistentDiskRequest'
+        appType:
+          type: string
+          description: currently only "GALAXY" is supported
+        kubernetesRuntimeConfig:
+          type: object
+          $ref: '#/components/schemas/KubernetesRuntimeConfig'
+        labels:
+          type: object
+          description: The labels to be placed on the cluster. Of type Map[String,String]
+
+    GetAppResponse:
+      description: the configuration of an app
+      type: object
+      properties:
+        kubernetesRuntimeConfig:
+          type: object
+          $ref: '#/components/schemas/KubernetesRuntimeConfig'
+        errors:
+          type: array
+          description: The list of errors that the app has encountered.
+          items:
+            $ref: "#/components/schemas/KubernetesError"
+        status:
+          type: string
+          $ref: "#/components/schemas/AppStatus"
+        proxyUrls:
+          type: object
+          description: map of service name to proxyUrl
+        diskName:
+          type: string
+          description: the name of the disk associated with this app
+
+    ListAppResponse:
+      description: the configuration of an app
+      type: object
+      properties:
+        googleProject:
+          type: string
+          description: the project this app is associated with
+        kubernetesRuntimeConfig:
+          type: object
+          $ref: '#/components/schemas/KubernetesRuntimeConfig'
+        errors:
```

Comment:
I think for `listRuntime`, we intentionally removed `errors` field from listRuntime response

Reply:
why? I think its helpful to include errors when listing

Reply:
reason back then was we weren't actually joining CLUSTER_ERROR table for listRuntime API, so it's always empty...And then we just decided to remove it, and also was leaning towards making list call more lean and having client to call `getRuntime` if they need to know about error

Reply:
To report the status for an app, we need to check if the cluster or nodepool for it has errored (i.e., if, during creation, we errorred before getting to app creation). So we already have to do the join,  I figure it can be helpful to return those errors